### DISCUSSION TITLE
[Q&A] make Password protected when uninstaling GLPI Agent

### DISCUSSION THREAD

**Discussion by dwiwibowo-003** [glpi-project/glpi-agent#523] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Is there a way to make GLPI-Agent is request password when uninstall its agent from add/remove program, so only IT can reinstall agent MSI. so we can make it security hardened

sorry if my question already answered.

**Comment by g-bougard** [comment]
Hi @dwiwibowo-003 

I'm not aware of such a feature on windows. GLPI-Agent must be installed with administrator role and so must be uninstalled with administrator role. I really don't know if it's possible to make such distinction into administrator roles.

Anyway this looks to me a windows OS problem. Is there any windows expert around to answer ?
